### PR TITLE
Update CONTRIBUTORS.csv

### PR DESCRIPTION
**Pull request by EliKalter**

### CODE REVIEW

File: CONTRIBUTORS.csv [hunk 1]
```diff
@@ -215,3 +215,4 @@ yes,jjdoar,Jacob Doar,<jjdoar@gmail.com>
 yes,saputkin,Arthur Saputkin,<saputkin@users.noreply.github.com> <artsapu@gmail.com>
 yes,talir0drigues,Talita Rodrigues,<trsantos92@gmail.com> <talita.rodrigues@ockam.io>
 yes,utkarshgupta137,Utkarsh Gupta,<utkarshgupta137@gmail.com>
+EliKalter, Eli Kalter, 02374elim@gmail.com
```
Comment (mrinalwadhwa) [inline CONTRIBUTORS.csv:218]: Thank you for sending this pull request.
If you're accepting our CLA. This line has to be similar to the previous line

```suggestion
yes,EliKalter,Eli Kalter,<02374elim@gmail.com>
```

Comment (EliKalter) [reply CONTRIBUTORS.csv:218]: > Thank you for sending this pull request. If you're accepting our CLA. This line has to be similar to the previous line

Thanks for clarifying, I thought if you approve me as a contributor the yes will be added on your side,
I'll fix it.

Comment (SanjoDeundiak) [reply CONTRIBUTORS.csv]: @EliKalter Could you please remove excess spaces to match the format of the rest of the file? Also, please squash your commits into 1

**Comment by EliKalter** [comment]
Sorry, I don't know how to squash the commits here and can't seem to find the settings for that,
I think it would be faster for me to open a new PR

**Pull request closed without merging**